ocaml/xenopsd/lib/xenopsd.ml

```diff
@@ -59,7 +59,7 @@ let feature_flags_path = ref "/etc/xenserver/features.d"
 
 let pvinpvh_xen_cmdline = ref "pv-shim console=xen"
 
-let numa_placement_compat = ref false
+let numa_placement_compat = ref None
```

Comment:
This could use a comment what the different values mean as we have None, Some(true), Some(false)

Reply:
Can't encode it as anything other than a boolean, unfortunately, because it needs to be backwards compatible (the print function is used in strange ways)

Reply:
I've added the code that decides the value depending on the old parameter right after the definition